addon/components/bread-crumbs.js

```diff
@@ -78,12 +78,16 @@ export default Component.extend({
   },
 
   _lookupBreadCrumb(routeNames, filteredRouteNames) {
-    const defaultLinkable = get(this, 'linkable');
+    let defaultLinkable = get(this, 'linkable');
+    const pathLength = routeNames.length;
     const breadCrumbs = filteredRouteNames.map((name, index) => {
       const path = this._guessRoutePath(routeNames, name, index);
       let breadCrumb = this._lookupRoute(path).getWithDefault('breadCrumb', undefined);
       const breadCrumbType = typeOf(breadCrumb);
 
+      if(index === pathLength - 1) {
+        defaultLinkable = false;
+      }
```

Comment:
I guess the CI failed due to space. Now it is cleared. But still the build fails in another test.